lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -445,20 +445,6 @@ ADDRESS_AND_FRAUD_J4:
     enhanced-verification:
       targetState: CRI_F2F
 
-# CRI escape journey (J5)
-CRI_DCMAW_J5:
```

Comment:
N.B. any removal or renaming of existing states _probably_ needs to be done in a two-step deployment, as it'll break any users in those states.

(Unless we're happy that this is a sufficiently small number of people)

Reply:
Thanks - if it helps, Chris has added a feature to the journey map which allows you to easily identify the orphaned states to clean up :)